transmission-remote-cli

```diff
@@ -1494,6 +1509,19 @@ class Interface(object):
                 server.remove_torrent_local_data(self.torrents[self.focus]['id'])
                 self.focus_next_after_delete()
 
+    def remove_finished_torrents(self, delete_local_data = False):
+        candidates = [torrent for torrent in self.torrents
+                if torrent['status'] == Transmission.STATUS_STOPPED
+                and torrent['uploadRatio'] >= torrent['seedRatioLimit']]
```

Comment:
I think this check should be omitted.  
People might want to remove all finished torrents regardless the seed ratio.